tracex/extraction/logic/modules/module_event_log_comparator.py

```diff
@@ -49,47 +53,44 @@ def __start_comparison(self, pipeline_df, ground_truth_df):
 
         with open(c.output_path.joinpath("event_log_comparison.txt"), "w") as f:
             f.write(
-                "Percentage of event information found by the pipeline that are contained in ground truth: "
+                "Percentage of activities found by the pipeline that are contained in ground truth: "
                 + str(matching_perc_pipeline_to_ground_truth)
-                + "\n"
+                + "%\n\n"
             )
             f.write(
-                "Percentage of event information in the ground truth that are contained event log by the pipeline: "
+                "Percentage of activities contained in ground truth that are found by the pipeline: "
                 + str(matching_perc_ground_truth_to_pipeline)
-                + "\n"
+                + "%\n"
             )
 
         return pipeline_df
 
     def __compare_activities(self, input_df, comparison_basis_df):
         total_matching_activities = 0
-        for activity in input_df["activity"]:
+        for i in range(len(input_df["activity"])):
             total_matching_activities += self.__find_activity(
```

Comment:
change to enumerate and rename variable i, please

Reply:
done